### DISCUSSION TITLE
Scheduler Suggestions

### DISCUSSION THREAD

**Discussion by nhodeisland** [SudoMagicCode/SudoSignals#10] [v1 - RFE & Feature Requests]
Often my clients want a TD project that has a bunch of generative or interactive content, some pre-made videos that the clients want to update all the time and they want all of that to be on a schedule.  

They also want to be able to integrate new rendered content (.mov files), and sometimes create their own preset looks using the parameters I have exposed to them.

My dream scheduler would do three things.

1) Allow my clients to play with the controls and save a preset, ideally with a text entry of some sort.
2) Schedule that preset on a timeline that looks similar to an iCal or Google Calendar (low learning curve)
3) ... this is the kicker, create a set of events that are the correct length as each video a folder.  This way I can schedule a set of sequential videos without having to put them all in a playlist and triggering the playlist.

**Comment by IanShelanskey** [comment]
Love these suggestions. Thanks @nhodeisland.

It sounds like you are looking for a way to hand-off installation controls and scheduling to the client. We are a bit far away from having a "client portal" available but have anticipated that folks would be interested in that. 

The roadmap currently has a scheduler with a drag-drop interface much like google calendar for a weekly timeline. Events on this calendar would allow you to set parameters that you can currently control in the dashboard.

Creating events based on videos in a folder is an interesting concept. We haven't mapped out a way for signals to be aware of assets/content on the local machine yet. But I'm assuming you want to have the lengths of every video accessible to the scheduler so you can reorder them however you want?
